sr_ethercat_hand_config/controls/host/lh/sr_edc_joint_position_controllers_PWM.yaml

```diff
@@ -42,43 +42,43 @@ sh_lh_lfj0_position_controller:
     i: 0.0
     i_clamp: 0.0
     max_force: 0.0
-    p: -4500.0
+    p: -4900.0
```

Comment:
Why do we have so different PID values for lh_LFJ0 and lh_FFJ0?

Reply:
As many hand we build, the parameters are more constant. It is because each robot we made we refine more the assembling processes and there are  more consistency between hands. Anyway, there is still a considerable uncertainty because of the chaos in the tendon routing under the wrist.
In this case, the difference on the FFJ0 and LFJ0 is because the routing under the palm, think about the fact that LFJ0 goes under joint LFJ5 as well, there are more curves in the tendon routing.

Reply:
OK